storageClient/storageClient.go

```diff
@@ -58,6 +58,25 @@ func (c *StorageClient) CreateGroup(ctx context.Context, params CreateGroupReque
 	return result, err
 }
 
+// getGroupByGroupNameAndAccountID fetches a group using the specified parameters,
+// returning the group and error (if any).
+func (c *StorageClient) getGroupByGroupNameAndAccountID(ctx context.Context, groupName string, accountID string) (*Group, error) {
+	var result *Group
+	path := c.Host + storageServiceBasePath + "/groups/" + groupName
+
+	request, err := e3dbClients.CreateRequest("GET", path, nil)
+	if err != nil {
+		return result, err
+	}
+	urlParams := request.URL.Query()
+	urlParams.Set("account_id", accountID)
+	urlParams.Set("group_name", groupName)
```

Comment:
group name should be in the path vs query, kinda like this
https://github.com/tozny/e3db-clients-go/blob/master/identityClient/identityClient.go#L661
except you need to url encode the group name as it could have spaces

Reply:
https://www.urlencoder.io/golang/#url-decoding-a-path-segment-in-golang

Reply:
Do I still keep group name here to be encoded?

Reply:
@galxy25 I think if a param needs to be url encoded it should live within the query params rather than the path

Reply:
I think I would side with levi here on keep it in the path. it allows for more expressive future endpoints

Reply:
But we might want to restrict (in code) what groupnames can be

Reply:
```
func (c *StorageClient) DescribeGroup(ctx context.Context, params DescribeGroupRequest) (*Group, error) {
	var result *Group
        groupNameURL:= url.Parse(fmt.Sprintf("/%s", params.Name))
	path := c.Host + storageServiceBasePath + "/groups/" + groupNameURL
	request, err := e3dbClients.CreateRequest("GET", path, params)
	if err != nil {
		return result, err
	}
	urlParams := request.URL.Query()
	urlParams.Set("account_id", params.AccountID.String())
	request.URL.RawQuery = urlParams.Encode()
	err = e3dbClients.MakeSignedServiceCall(ctx, request, c.SigningKeys, c.ClientID, &result)
	return result, err
}
```

Reply:
Would the encoding part look like this?

Reply:
@rosasck use https://golang.org/src/net/url/url.go?s=7851:7884#L264 QueryEscape to encode the params.name as part of the path. Use on the server side when parsing this value from the path https://golang.org/pkg/net/url/#QueryUnescape